### DISCUSSION TITLE
[Feature Request] ​Allow text to align right, center

### DISCUSSION THREAD

**Discussion by xvzhenduo** [toeverything/AFFiNE#8617] [Ideas]
### Description

Allow text to align right, center

### Use case

When i'm writing a letter, the signature should be right-aligned. Sometimes the header look better when aligned in the center

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by hwangdev97** [comment]
We should support this feature, including image alignment! It might take a little time to get everything ready, though. Right now, we're concentrating on performance optimization, mobile , and the database. Thanks for your patience!
